src/components/Alert/Alerts.tsx

```diff
@@ -0,0 +1,28 @@
+import React, {useState} from 'react';
```

Comment:
This file should be named  `AlertNotification.tsx` and `AlertNotification.module.scss` to avoid conflicting with the built in Alert component